### PR TITLE
Update ReadMe.md

### PR DESCRIPTION
**Pull request by alistair23**

### CODE REVIEW

**Comment by review-ninja** [comment]
<a href="https://app.review.ninja/alistair23/ReviewNinja-Welcome/pull/1" target="_blank">ReviewNinja

File: ReadMe.md [hunk 1]
```diff
@@ -1 +1,3 @@
-# ReviewNinja-Welcome
\ No newline at end of file
+# ReviewNinja-Welcome
```
Comment (alistair23) [inline ReadMe.md]: Note

Comment (alistair23) [inline ReadMe.md]: !fix Fix this

Comment (alistair23) [inline ReadMe.md]: !fixed This is now fixed. Good job

**Pull request merged**